automatic/minikube/README.md

```diff
@@ -0,0 +1,15 @@
+# <img src="https://cdn.rawgit.com/abejenaru/chocolatey-packages/16e4ddf69bd7dfc07597b7b3aaa7d8372eeec7fb/icons/minikube.png" width="48" height="48"/> [minikube](https://chocolatey.org/packages/minikube)
+
+Minikube is a tool that makes it easy to run Kubernetes locally.
+
+Minikube runs a single-node Kubernetes cluster inside a VM on your laptop for users looking to try out Kubernetes or develop with it day-to-day.
+
+### Minikube Features
+Minikube supports Kubernetes features such as:
+* DNS
+* NodePorts
+* ConfigMaps and Secrets
+* Dashboards
+* Container Runtime: Docker, rkt and CRI-O
+* Enabling CNI (Container Network Interface)
+* Ingress
```

Comment:
Please add a notes section that mentions that the software is only for 64bit systems.